stdlib/n-ary-app.kl

```diff
@@ -105,10 +105,17 @@
                     [(list (arg1 arg2))
                      (pure (list-syntax ('-> arg1 arg2)
                                         stx))]
+                    [(list (arg1))
+                     (pure (list-syntax ('-> arg1)
+                                        stx))]
```

Comment:
The kernel now supports 0, 1, or 2 arguments (`(->)`, `(-> A)`, or `(-> A B)`). The prelude's n-ary wrapper was only supporting 2 or more arguments. This extends the wrapper to also support 1 argument.